src/Tribe/Importer/File_Importer_Organizers.php

```diff
@@ -37,17 +37,18 @@ protected function create_post( array $record ) {
 	}
 
 	/**
-	 * Build a organizer array for creation/update of the current imported organizer.
+	 * Build an organizer array for creation/update of the current imported organizer.
 	 *
 	 * @since 3.2
 	 * @since 5.1.6 Adjust to prevent overwriting values that aren't mapped.
+	 * @since TBD   Add code to allow importing custom data.
 	 *
 	 * @param int   $organizer_id The ID of the organizer we're currently importing.
 	 * @param array $record       An event record from the import.
 	 *
 	 * @return array $organizer The array of organizer data for creation/update.
 	 */
-	private function build_organizer_array( $organizer_id, array $record ) {
+	private function build_organizer_array( int $organizer_id, array $record ): array {
```

Comment:
The filter `tribe_events_importer_organizer_fields` is already released for a while. That can modify the return type of the method, so i am a little concerned of enforcing `array` as the return type.

Reply:
Removed return type forcing.